Airbyte Hubspot Error:  bufio.Scanner: token too long.

## Environment
Deploy method: jitsucom/jitsu docker
Jitsu version: for instance 1.39.1
OS: Ubuntu 20.04
Cloud Provider: AWS EC2

## Description
The synchronisation of the engagement stream of the Hubspot airbyte source to a postgres destination fails with: 
```
2022-02-09 00:00:45 [DEBUG] - {"type": "LOG", "log": {"level": "INFO", "message": "Syncing engagements stream"}}
2022-02-09 00:00:49 [INFO ] - Stream [campaigns] Table name [hubspot_campaigns] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Synchronized successfully Table [hubspot_campaigns] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Stream [contacts_list_memberships] Table name [hubspot_contacts_list_memberships] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Synchronized successfully Table [hubspot_contacts_list_memberships] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Stream [deal_pipelines] Table name [hubspot_deal_pipelines] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Synchronized successfully Table [hubspot_deal_pipelines] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Stream [engagements] Table name [hubspot_engagements] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Synchronized successfully Table [hubspot_engagements] key fields [] objects [NUMBER]
2022-02-09 00:00:49 [INFO ] - Stream [contacts] Table name [hubspot_contacts] key fields [] objects [NUMBER]
2022-02-09 00:00:50 [INFO ] - Synchronized successfully Table [hubspot_contacts] key fields [] objects [NUMBER]
2022-02-09 00:00:50 [INFO ] - Stream [email_events] Table name [hubspot_email_events] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Synchronized successfully Table [hubspot_email_events] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Stream [companies] Table name [hubspot_companies] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Synchronized successfully Table [hubspot_companies] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Stream [deals] Table name [hubspot_deals] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Synchronized successfully Table [hubspot_deals] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Stream [contact_lists] Table name [hubspot_contact_lists] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [INFO ] - Synchronized successfully Table [hubspot_contact_lists] key fields [] objects [NUMBER]
2022-02-09 00:00:51 [ERROR] - Process error: bufio.Scanner: token too long. Process will be killed
2022-02-09 00:01:51 [ERROR] - Error closing airbyte runner: exit status 1
2022-02-09 00:01:51 [ERROR] - Error synchronization: signal: killed
```

Could this be a issue of jitsu or does it come from airbyte?

--- a/server/airbyte/async_parser.go
+++ b/server/airbyte/async_parser.go
@@ -8,6 +8,7 @@ import (
 	"github.com/jitsucom/jitsu/server/logging"
 	"github.com/jitsucom/jitsu/server/schema"
 	"io"
+	"math"
 )
 
 const (
@@ -43,7 +44,7 @@ func (ap *asynchronousParser) parse(stdout io.Reader) error {
 
 	scanner := bufio.NewScanner(stdout)
 	buf := make([]byte, 0, 64*1024)
-	scanner.Buffer(buf, 1024*1024)
+	scanner.Buffer(buf, math.MaxInt/2)
 
 	records := 0
 	for scanner.Scan() {

--- a/server/airbyte/runner.go
+++ b/server/airbyte/runner.go
@@ -169,7 +169,7 @@ func (r *Runner) Read(dataConsumer base.CLIDataConsumer, streamsRepresentation m
 
 		err := asyncParser.parse(stdout)
 		if err != nil {
-			taskCloser.CloseWithError(fmt.Sprintf("Process error: %v. Process will be killed", err), false)
+			taskCloser.CloseWithError(fmt.Sprintf("Airbyte output line process error: %v. Process will be killed", err), false)
 			if killErr := r.Close(); killErr != nil && killErr != runner.ErrAirbyteAlreadyTerminated {
 				taskLogger.ERROR("Error closing airbyte runner: %v", killErr)
 				logging.Errorf("[%s] closing airbyte runner: %v", taskCloser.TaskID(), killErr)

--- a/server/drivers/singer/parser.go
+++ b/server/drivers/singer/parser.go
@@ -10,6 +10,7 @@ import (
 	"github.com/jitsucom/jitsu/server/schema"
 	"github.com/jitsucom/jitsu/server/singer"
 	"io"
+	"math"
 )
 
 const (
@@ -47,7 +48,7 @@ func (sop *streamOutputParser) Parse(stdout io.ReadCloser) error {
 
 	scanner := bufio.NewScanner(stdout)
 	buf := make([]byte, 0, 64*1024)
-	scanner.Buffer(buf, 1024*1024)
+	scanner.Buffer(buf, math.MaxInt/2)
 
 	records := 0
 	streamCleaned := map[string]bool{}

